Source/Krypton Components/Krypton.Ribbon/Palette/ContextTabSet.cs

```diff
@@ -125,7 +125,7 @@ public Point GetRightScreenPosition()
         /// <summary>
         /// Gets the name of the context.
         /// </summary>
-        public Color ContextColor => Context.ContextColor;
+        public Color ContextColor => ((Color)Context.ContextColor)!;
```

Comment:
`Color` should never be null, change code to init and use `color.empty` for checks